client/app/telecom/pack/xdsl/modem/service/modal/pack-xdsl-modem-service-modal.html

```diff
@@ -0,0 +1,12 @@
+<oui-modal data-heading="{{ 'xdsl_modem_service_modal_title' | translate }}"
+           data-primary-action="$ctrl.close()"
+           data-primary-label="{{:: 'common_confirm' | translate }}"
+           data-secondary-label="{{:: 'cancel' | translate }}"
+           data-secondary-action="$ctrl.cancel()"
+           data-on-dismiss="$ctrl.cancel()"
+           data-type="warning">
+    <div class="text-warning mt-3">
```

Comment:
No, this modal is displayed to inform the customer that it will disable several elements by its action (and as we are at the bottom of the page if we use a <oui-message> it will be displayed at the very top of the page. page, what we do not want).

Reply:
He meant you can replace the `div` by a `oui-message`, you'll still be in a modal :)